superset/reports/commands/execute.py

```diff
@@ -352,11 +366,18 @@ def _get_notification_content(self) -> NotificationContent:
                     error_text = "Unexpected missing screenshot"
             elif (
                 self._report_schedule.chart
-                and self._report_schedule.report_format == ReportDataFormat.DATA
+                and self._report_schedule.report_format == ReportDataFormat.CSV
```

Comment:
Add `table_like` method for `ReportDataFormat` and use the next condition:
```python
if self._report_schedule.report_format in ReportDataFormat.table_like():
```

For `ReportDataFormat`:
```python
@classmethod
def table_like(cls) -> Set["ReportDataFormat"]:
    return {cls.CSV} | {cls.XLSX}
```